.github/workflows/full-android.yaml

```diff
@@ -16,6 +16,7 @@ name: Full builds - Android
 
 on:
     push:
+    pull_request:
```

Comment:
This was excluded purposefully, last I checked.  Please make sure @andy31415 reviews this.

Reply:
Done